app/ide-desktop/lib/dashboard/src/authentication/src/authentication/components/login.tsx

```diff
@@ -0,0 +1,32 @@
+/** @file Login container responsible for rendering and interactions in sign in flow. */
+import * as router from "react-router-dom";
+
+import withRouter from "../../navigation";
+import * as app from "../../components/app";
+import * as icons from "../../components/svg";
+
+// ======================
+// === loginContainer ===
+// ======================
+
+const loginContainer = () => {
+  return (
```

Comment:
i'm guessing this file (as well as the rest) should be using 4 spaces like most of the other files?

Reply:
Actually I don't know if our indentation is standardized anywhere. I think I've mostly been using 4 spaces but prettier has changed it in some places. I'll change this file to 4.

Reply:
btw: if you go to the "files changed" tab, i think most of the files below this one also have 2-space indentation

Reply:
@somebody1234 solved?

Reply:
@PabloBuchu @somebody1234 going to address the spacing & indentation after these PRs are merged as the number of merge conflicts introduced by this change is significant.